R_demos/4-exomeseq-introduction.Rmd

```diff
@@ -0,0 +1,267 @@
+---
+title: "Genomic Variants: Introduction to exomeSeq data"
+author: Jineta Banerjee
+date: '`r format(Sys.time(), "%d %B, %Y")`'
+output: 
+  html_document:
+    code_folding: hide
+    toc: true
+    number_sections: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    fig_width: 7
+    fig_height: 6
+    fig_caption: true
+    df_print: paged 
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Getting Started
+
+This document aims to guide the Research Case participants, who have limited exposure to Synapse as a platform and clinical/biological data, through exploration and analysis of the available exomeSeq datasets. The available dataset has been carefully curated to include genomic variant (exomeSeq) data from normal and three different tumor types from two different study initiatives into a single platform (a harmonized dataset). 
```

Comment:
we might need to update `Research Case` to `Hackathon` at some point. @allaway have we come to a conclusion about this?

Reply:
I already pushed a change to reflect hackathon after seeing the messages on SVAI slack... wonder why it is not reflected here ... :(

Reply:
Yep, hackathon is the term. looks like this was already changed :)